readyapi4j-groovy-dsl/src/main/groovy/com/smartbear/readyapi4j/dsl/pro/ProDslDelegate.groovy

```diff
@@ -0,0 +1,29 @@
+package com.smartbear.readyapi4j.dsl.pro
+
+import com.smartbear.readyapi4j.dsl.DslDelegate
+
+/**
+ * The delegate responding to commands inside the "recipe" closure of ServerTestDsl.
+ */
+class ProDslDelegate extends DslDelegate {
```

Comment:
It'd be nice to introduce local pro support pretty soon. Shouldn't be difficult. We just want to make sure that the right libraries are loaded. License checks will be performed by the Pro core.